Update legislation.tex

mep-clearhaus opened the pull request:
New and up-to-date version of the cake legislation.

OBS, I need someone to be create a PDF version of this whenever merged.

New stuff
- Added both Quickpay and 3DSecure.io, and their employees, as potential COR. 
- Removed item relating to Dinero
- Added "Size"/"Amount" in relation *Cake*
- Adding a rule relating to the POP2 Padel Court
- Removing superfluous paragraphs to ensure we are being served a little less cake than the old legislation was permitting

mep-clearhaus committed:
Update legislation.tex

New and up-to-date version of the cake legislation.

New stuff
- Added both Quickpay and 3DSecure.io, and their employees, as potential COR. 
- Removed item relating to Dinero
- Added "Size"/"Amount" in relation *Cake*
- Adding a rule relating to the POP2 Padel Court
- Removing superfluous paragraphs to ensure we are being served a little less cake than the old legislation was permitting

ct-clearhaus commented on legislation.tex:
```diff
@@ -29,8 +29,8 @@
 \begin{enumerate}[§ 1]
-  \item This legislation applies to all employees of Clearhaus {\small A/S} and
-    any subsidiary companies in Denmark, as well as abroad.
+  \item This legislation applies to all employees of Clearhaus, Quickpay, and 3DSecure.io.
```
```suggestion
  \item This legislation applies to all employees of Clearhaus, Quickpay, and 3dsecure.io.
```

Capitalisation cf. https://datacvr.virk.dk/enhed/virksomhed/41019328.

ct-clearhaus commented on legislation.tex:
```diff
@@ -68,16 +70,13 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
   \item Winning a programming competition.
   \item Returning from parental leave.
-  \item Full-time employment in Clearhaus A/S or any other company.
+  \item Going from student position, internship or similar non-full-time-employment to Full-time employment in Clearhaus, Quickpay, or 3DSecure.io.
```
```suggestion
  \item Going from student position, internship or similar non-full-time-employment to full-time employment in Clearhaus, Quickpay, or 3dsecure.io.
```

Consequence.

ct-clearhaus reviewed (dismissed):
Nice! 🎾

Should we add also 🏓 while we're at it? 🤔

mep-clearhaus commented:
> Should we add also 🏓 while we're at it? 🤔

Yes!

mep-clearhaus committed:
Update legislation.tex

Added a new Item relating to table tennis

mep-clearhaus committed:
Apply suggestions from code review

Co-authored-by: Casper Thomsen <ct-clearhaus@users.noreply.github.com>

cbobach commented:
We were considering if we should add that `Clearhaus gives cake if network is down` 🤷🏻

brorbw commented:
Something like `If wifi down > 15 min CH will provide cake`. This will also make up for missed cake due to removal of sections

Lassejoe commented:
> wifi down > 15 min

Or `if Wi-Fi down multiple times same day`
Or `if Wi-Fi down due to new settings`

mep-clearhaus committed:
Update legislation.tex

Added thoughts around wifi disturbances. Does it fly, or should it be thrown into two different items? My idea was that cake should not be offered twice if wifi is down for more than 15 minutes, and it goes down for 1 minuter later the same day. WDYT?

mep-clearhaus commented:
I've tried to add the wifi disturbance input :D Please check it out

cbobach commented on legislation.tex:
```diff
@@ -83,5 +83,6 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
   \item Being made (partially) superfluous by automation.
   \item Loosing office key fob.
   \item Occupying a meeting room booked by another person (without leaving immediately)
+  \item If wifi down > 15 min, or/and wifi down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake
```
```suggestion
  \item If network down > 15 min, or/and network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake
```

cbobach reviewed (commented):
Suggestion 🤷🏻

mep-clearhaus committed:
Update legislation.tex

Co-authored-by: Christian Bobach <christian.bobach@gmail.com>

kbadk commented on legislation.tex:
```diff
@@ -68,20 +70,19 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
   \item Forgetting that you are on call.
   \item Being made (partially) superfluous by automation.
   \item Loosing office key fob.
   \item Occupying a meeting room booked by another person (without leaving immediately)
```
```suggestion
  \item Occupying a meeting room booked by another person (without leaving immediately).
```

kbadk commented on legislation.tex:
```diff
@@ -68,20 +70,19 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
   \item Being made (partially) superfluous by automation.
   \item Loosing office key fob.
   \item Occupying a meeting room booked by another person (without leaving immediately)
+  \item If network down > 15 min, or/and network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake
```
```suggestion
  \item If network down > 15 min, and/or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
```

cbobach committed:
Update legislation.tex

Co-authored-by: Kristian Antonsen <kristian@derfor.dk>

cbobach committed:
Update legislation.tex

Co-authored-by: Kristian Antonsen <kristian@derfor.dk>

kbadk committed:
Merge branch 'master' into master

brorbw commented on legislation.tex:
```diff
@@ -83,6 +83,6 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
   \item Loosing office key fob.
   \item Occupying a meeting room booked by another person (without leaving immediately).
-  \item If network down > 15 min, or/and network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake
+  \item If network down > 15 min, and/or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
```
`or` should suffice. Its ambiguous if both conditions should apply /s
```suggestion
  \item If network down > 15 min, or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
```

cbobach committed:
Update legislation.tex

Co-authored-by: Bror <brorwinther@gmail.com>
